### ISSUE TITLE
安装好了，启动报错

### ISSUE THREAD

**Issue by mountainguan** [Yufeikang/bcloud#30]
** (bcloud-gui:20811): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-KSMm7djLmn: 拒绝连接
Traceback (most recent call last):
  File "/usr/local/bin/bcloud-gui", line 11, in <module>
    from bcloud.App import App
  File "/usr/local/lib/python3.5/dist-packages/bcloud/App.py", line 50, in <module>
    gi.require_version('AppIndicator3', '0.1')
  File "/usr/lib/python3/dist-packages/gi/__init__.py", line 118, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace AppIndicator3 not available

**Comment by Yufeikang** [comment]
只支持gtk平台

**Comment by Yufeikang** [comment]
手动安装 libappindicator-gtk3 试试

**Comment by windilt** [comment]
同样问题，opensuse平台没有 libappindicator-gtk3 包。。。

S  | 名称                          | 摘要                                             | 类型  
---+-------------------------------+--------------------------------------------------+-------
i  | libappindicator3-1            | Application indicators library -- GTK+ 3         | 软件包
i+ | libappindicator3-1-debuginfo  | Debug information for package libappindicator3-1 | 软件包
i+ | libappindicator3-devel        | Development files for libappindicator3           | 软件包
   | typelib-1_0-AppIndicator3-0_1 | Application indicators library                   | 软件包
